### DISCUSSION TITLE
Awesome List

### DISCUSSION THREAD

**Discussion by alecloudenback** [actuarialopensource/website#32] [Team Posts]
Open source has a [long tradition](https://github.com/sindresorhus/awesome) of "awesome lists", which are basically simple organized lists to point to other items of interest. I created [awesome-actuarial](https://github.com/actuarialopensource/awesome-actuarial) with some suggested sections, formatting, and a sampling (for sure not complete) of links. Please contribute! 

I'm not committed to a single structure or fixed set of sections. If you have ideas for a major overhaul, open an issue or let's discuss here. Otherwise, feel free to make incremental edits anytime.

**Comment by fumitoh** [comment]
Done

**Comment by jbogaardt** [comment]
Although he doesn't call it an awesome list, here is a similar list:
https://github.com/genedan/actuarial-foss

**Comment by fumitoh** [comment]
@jbogaardt Thank you! I just invited him.
